pages/about/index.tsx

```diff
@@ -23,118 +42,134 @@ const AboutPage = () => {
               className="crop-photo rounded-3xl"
             />
           </div>
+        </div>
 
-        <MaxWidth>
-          <div className='grid grid-flow-col grid-rows-3'>
-            <div className="flex items-center justify-center">
-              <h1 className=" hero-xl">About Us.</h1>
-            </div>
-
-            <div>
-              <h2 className="mb-3 text-h2 font-bold text-white">
-                <div className='flex items-center'>
-                  <div className='mr-2'>
+        <div className='flex justify-center sm:mt-28'>  
+          <Box boxShadow='2xl' p='6' rounded='xl' bg='white'
+            width={{base: '360px', md: "90%", xl: '95%'}}
```

Comment:
1. Can the base be narrower as this is causing the card to occupy the whole width at 360px without margin? Could be done as fixed value or maybe view width percentage
2. Can we also have breakpoint at `sm` as the current tabs overflow from `sm` to `md`? 
![image](https://user-images.githubusercontent.com/8915196/205679931-e0793cf2-8abc-4a7f-9f0f-16c825c0e5a6.png)